MekHQ/resources/mekhq/resources/GMToolsDialog.properties

```diff
@@ -0,0 +1,38 @@
+# GM Tools Dialog Resource Strings
+# Dice Panel
+dicePanel.text=Dice Roller

```

Comment:
Weird line endings need to be resolved.

Reply:
Carriage returns, apparently. Will clean them out once I finish my current option swapover.

Reply:
Resolved, but had to add the Bloodname Panel start to get it to be picked up (I'm planning on handling that immediately, so no issues with it being left behind)